servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -2011,7 +2011,8 @@ void RenderForwardClustered::_render_scene(RenderDataRD *p_render_data, const Co
 		}
 		if (needs_pre_resolve) {
 			//pre clear the depth framebuffer, as AMD (and maybe others?) use compute for it, and barrier other compute shaders.
-			RD::get_singleton()->draw_list_begin(depth_framebuffer, RD::DRAW_CLEAR_ALL, depth_pass_clear, 0.0f);
+			Rect2i region = RendererRD::TextureStorage::get_singleton()->render_target_get_override_render_region(rb->get_render_target());
```

Comment:
We should clean this up to consider the case where there is no render target override. Most of the time when this code runs, it will be in a non-XR environment, so we shouldn't add something that assumes we are in XR _and_ using dynamic scaling.

Reply:
I've updated it so that `render_region` is no longer a part of the override/xr stuff, it's now just a property of the `RenderTarget` struct.